apps/server/src/modules/authorization/domain/rules/board-do.rule.ts

```diff
@@ -16,30 +23,84 @@ export class BoardDoRule implements Rule {
 
 	public hasPermission(user: User, boardDoAuthorizable: BoardDoAuthorizable, context: AuthorizationContext): boolean {
 		const hasPermission = this.authorizationHelper.hasAllPermissions(user, context.requiredPermissions);
-		if (hasPermission === false) {
+		if (!hasPermission) {
 			return false;
 		}
 
-		const userBoardRole = boardDoAuthorizable.users.find(({ userId }) => userId === user.id);
-		if (!userBoardRole) {
+		const userBoardRoles = boardDoAuthorizable.users.find(({ userId }) => userId === user.id);
+		if (!userBoardRoles) {
 			return false;
 		}
 
-		if (boardDoAuthorizable.requiredUserRole && boardDoAuthorizable.requiredUserRole !== userBoardRole.userRoleEnum) {
+		// permission< for elements under a submission item are handled by the parent submission item
+		if (isSubmissionItem(boardDoAuthorizable.parentDo)) {
```

Comment:
correct, this is not designed to ask for down the tree, but is only for the specific case of elements inside the submission items, one level. For now, the logic is the same, code is only refactored to be moved into authrozation instead of in many places in the use cases